lib/cancan/model_adapters/sti_normalizer.rb

```diff
@@ -0,0 +1,32 @@
+# this class is responsible of normalizing the hash of conditions
```

Comment:
I think this part of comments should be removed. it has been copy-pasted from the rules normalizer.